sklearn/decomposition/tests/test_nmf.py

```diff
@@ -121,6 +121,18 @@ def test_nmf_transform():
 
 
 @ignore_warnings
+def test_nmf_inverse_transform():
+    # Test that NMF.inverse_transform returns close values
+    A = np.abs(random_state.randn(6, 4))
```

Comment:
It's really ugly that we are using a random_state global to the module, and not local to each test (I know that you are not responsible for that).

It means the tests will depend on the order in which they are executed.

Ideally, it would be great if you could add at the beginning of your test: "random_state = np.random.RandomState(0)" to avoid that problem.